packages/api-reference/src/features/example-responses/ExampleResponses.vue

```diff
@@ -67,42 +69,46 @@ const currentJsonResponse = computed(() => {
     normalizedContent?.['*/*'] ??
     // Take the first key - in the future we may want to use the selected content type
     normalizedContent?.[keys[0]] ??
-    // Swagger 2.0
-    currentResponse.value
+    undefined
   )
 })
 
+const hasMultipleExamples = computed<boolean>(
+  () =>
+    !!currentResponseContent.value?.examples &&
```

Comment:
```suggestion
    Boolean(currentResponseContent.value?.examples) &&
```
Do you like the way boolean looks?